LICENSE

```diff
@@ -1,5 +1,9 @@
 AWS JDBC Driver for PostgreSQL and all Amazon modifications to PostgreSQL JDBC Driver
-Copyright Amazon.com Inc. or affiliates.
+Copyright Amazon.com Inc. or affiliates. All rights reserved.
+
+PostgreSQL JDBC Driver - https://jdbc.postgresql.org/
+Copyright (c) 1997, PostgreSQL Global Development Group
+All rights reserved.
```

Comment:
The usual way in these things is to leave the original copyright notice at the top and then add 

Partial Copyright (c) 2021 Amazon ...

second.

Reply:
If I was forking a file, I'd agree. Original file's copyright first, then modifications after that; giving a hint to the history of the file.

With a project though, I like to see the project 'owner' first, and then other copyright after that. Helping to identify what the project is and helping to direct communication (be it happy or sad :) ) to the forked project owner instead of to the original project's community. 

Just to explain my thinking :) If the Postgresql project would prefer it the other way, then happy to adjust.

Reply:
@hyandell So I'm not quite sure why you would not fork the project? 

So my challenge when I see the copyright the way it is written is that it appears that this new project is the copyright owner for the entire codebase. Because of the way the LICENSE file is referred to in all of the existing files you are now exerting copyright over all of the files which I see as being incorrect. I'm not sure how to resolve this at the moment but my understanding is that you own the copyright only on the files that you have added and then partial copyright on anything you have modified.

Reply:
Henry,

neither Dave nor I can speak for the PostgreSQL project, even though I was for some 10 years a member of the PostgreSQL core team. If broader feedback and community input is desired, we should move this discussion to the appropriate [mailing list](https://jdbc.postgresql.org/community/community.html).

Reply:
@davecramer how would you expect a forked project to look? My concern with our current approach of a separate license file and third-party-license file is that it's not giving the PostgreSQL licensing enough kudos, so I looked to clarify by bringing them together. Typically I like a license file to cover the main project's core licensing, while any third-party-license filing is more ancillary, be it dependencies or separate contrib packages.

As you said, the copyright on the project is a mix of original PostgreSQL community licensing, and subsequent AWS community changes. 

@wieck understood - very happy to have both of your feedback though. I should also point out I'm not a project member of this repo either, just a nosy OSPO team member :)

Reply:
@hyandell I guess I would have expected you to actually fork it on github so that you could do pull requests from it.

Reply:
Ohh. GitHub-fork. I understand now. Sorry, I thought you were using a more generalized 'fork' meaning, such as 'has some code that's unlikely to be accepted upstream'.